### DISCUSSION TITLE
Question on Harvesters

### DISCUSSION THREAD

**Discussion by Alan911Black** [Chia-Network/chia-blockchain#17232] [Q&A]
i have a main Harvester and 1 Remote. what happened if the main Harvester goes down, will the Remote will still get plot challenges?

**Comment by BrandtH22** [comment]
Hey @Alan911Black  , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

If you have followed the setup here https://docs.chia.net/farming-on-many-machines/ then the remote harvester requires the main node to provide partials and to sign the actual blocks. This means that the main node going down will lead to the remote harvester no longer farming.
